src/code/utilities/guide-protocol.js

```diff
@@ -40,14 +40,24 @@ GuideProtocol.Event = function(username, session, sequence, actor, action, targe
 
 GuideProtocol.Event.Channel = 'Event';
 
+GuideProtocol.Event.prototype.isMatch = function(actor, action, target) {
```

Comment:
I don't understand what the purpose of this function is, but this all looks good otherwise!